app/views/layouts/application.html.erb

```diff
@@ -11,6 +11,21 @@
   </head>
 
   <body>
-    <%= yield %>
+  <div class="container">
```

Comment:
Line 15 - 25  will be better if you use a partial

Reply:
oke sis, I will practice using partial